checkov/gitlab_ci/image_referencer/base_provider.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+from typing import Any
+
+from checkov.common.images.image_referencer import Image
+
+
+class BaseGitlabCiProvider:
```

Comment:
```suggestion
class GitlabCiProvider:
```
I think you don't need to name it `Base...` because you don't subclass it somewhere else 🙂 you can also change the name just to `provider.py`